lib/wings/orm_converter.rb

```diff
@@ -55,6 +55,14 @@ def self.to_s
           internal_resource
         end
 
+        def to_model
+          Wings::ActiveFedoraConverter.convert(resource: self)
+        end
+
+        def errors
```

Comment:
This is for when #errors is called on the JSON response. Allows the use of the #add method for the errors to be appended.

Reply:
i'm wondering how this works? does it return the correct error object?